### PR TITLE
Add tests for all the key code constants

### LINKED ISSUE #4: Write unit tests to test it returns all the constant keys with correct values
Write unit tests to test it returns all the constant keys with correct values

### PR DESCRIPTION
**Pull request by ka3work**
Fixes #4 

Currently covers all of the 115 consts.

Please let me know if you would like to see any changes. :grinning:

### CODE REVIEW

**Comment by kabirbaidhya** [comment]
@frameshft Great, it looks good to me. Thanks for contributing.

> Currently covers all of the 115 consts.

This seems like a tedious thing BTW :smile: :+1: 

Anyway, I'll integrate Travis, check these in the CI and merge it.

**Comment by ka3work** [comment]
@kabirbaidhya Glad to help :+1: 

I just noticed that I've made a typo and tested KEY_Y with KEY_X.

Made an additional commit, or you would like me to squash it?

**Comment by kabirbaidhya** [comment]
@frameshft It's okay, not a big deal. The main thing is that we need to verify these values against [this](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode#Constants_for_keyCode_value) which is a kind of spec that browsers follow.

File: test/KeyCode.spec.js [hunk 1]
```diff
@@ -48,4 +48,463 @@ describe('KeyCode', () => {
         assert(keyPressed === KeyCode[randomItem.name]);
     });
 
+    it('KEY_CANCEL should return 3', () => {
```
Comment (kabirbaidhya) [inline test/KeyCode.spec.js]: @frameshft Can we change the wording `should return` to `equals` and use it for all these tests? Something like this `KEY_CANCEL equals 3`?

**Review by kabirbaidhya — CHANGES_REQUESTED** [review]

**Comment by ka3work** [comment]
@kabirbaidhya Done :+1:

**Comment by kabirbaidhya** [comment]
@frameshft I've made some changes and Integrated Travis CI, see https://github.com/kabirbaidhya/keycode-js/pull/11. Can you pull the latest changes and push again?

**Comment by ka3work** [comment]
@kabirbaidhya Alright I've made a quick rebase, please review the changes.

**Review by kabirbaidhya — APPROVED** [review]
Looks good to me :+1:

**Review by mesaugat — APPROVED** [review]

**Pull request merged**